lib/ui/dashboard.js

```diff
@@ -779,9 +779,49 @@ class Dashboard {
       updatePodsTable();
       screen.render();
     }
+
+    setupToggle(screen, debug);
   }
 }
 
+function setupToggle (screen, debug) {
+  const validElementValues = Object.values({
+    logs: "Logs",
+    pods: "Pods",
+  });
+
+  screen.key(['tab', 'S-tab'], function (_, key) {
```

Comment:
Should that listener be removed when moving away from the dashboard tab? If yes, it is possible to use the navbar events, .e.g.:

https://github.com/astefanutti/kubebox/blob/1e599a10d52e32cb47cec7fca90e519779aaa251/lib/ui/events.js#L139

Reply:
I don't think it is necessary, have tested this a lot, had a `debug.log(...)` in this event. When I press the `tab` key in the debug view nothing happens.

Reply:
OK, this can be done later if needed then.